utils/common-messages.ts

```diff
@@ -7,6 +7,10 @@ export const commonMessages = defineMessages({
     id: 'button.cancel',
     defaultMessage: 'Cancel',
   },
+  dateAtTimeTooltip: {
+    id: 'tooltip.date_at_time',
```

Comment:
```suggestion
    id: 'tooltip.date-at-time',
```

NIT: key consistency